content/posts/newsletter/2021-07-05.md

```diff
@@ -0,0 +1,184 @@
+
+Vaxbot, the XMPP based vaccine bot [has been featured on Canadian National News](https://monal.im/blog/vaxbot-on-canadian-national-news/).
+
+Software freedom conservancy [moved their chat from IRC to XMPP](https://sfconservancy.org/blog/2021/jun/21/xmpp-chat/) using [Snikket](https://snikket.org/).
+
+Ingrid’s Space has thoughts on [why decentralised applications don’t work](https://ingrids.space/posts/why-distributed-systems-dont-work/).
+
+## Software news
+
+### Clients and applications
+
+UWPX [v.0.33.0.0](https://github.com/UWPX/UWPX-Client/releases/tag/v.0.33.0.0) has been released. It includes support for [XMPP Providers](https://invent.kde.org/melvo/xmpp-providers/-/tree/master) for convenient auto-complete and registration of new accounts to XMPP servers. Besides that, this release fixes a few bugs related to stream close and open message parsing.
+![UWPX](https://user-images.githubusercontent.com/11741404/124069653-cf8dc600-da3c-11eb-86b7-324cbc6ac26f.png)
+
+[Gajim Development News](https://gajim.org/post/2021-06-27-development-news-june/): This month brought some fixes for themes and spell checking. 
+Development on the new Gajim version made a big step forward: file previews are working again and more important they're now a core integrated feature, they look even better while also being privacy respecting for those anonymous chats.
+
+Monal 5.0 [has been released](https://monal.im/blog/monal-with-group-chat-and-voice-messages/), featuring support for group chats (not yet OMEMO encrypted, will come next) and sending voice messages and videos, but also a lot of internal code refactoring and bug fixes. 
+![Monal](https://monal.im/wp-content/uploads/2021/06/Monal1.png)
+
+Blabber.IM XMPP service has been discontinued and has shut down its server. On June 25, the official blabber.im website reported that they were discontinuing support for their XMPP server.
+
+Profanity, a console-based XMPP client, has implemented [attention flags](https://profanity-im.github.io/blog/post/attention-attention/). Use the shortcut ALT+F to toggle the flag.
+
+## Servers
+
+[Jackal](https://github.com/ortuman/jackal), an XMPP server written in Go, released [version 0.54.0](https://github.com/ortuman/jackal/releases/tag/v0.54.0) with support for [XEP-0198: Stream Management](https://xmpp.org/extensions/xep-0198.html), docker compose support, graphana service dashboard and a new fancy logo:
+
+![New fancy Jackal logo](https://raw.githubusercontent.com/ortuman/jackal/master/logos/logo-0.png)
+
+[OpenFire 4.6.4 has been released](https://discourse.igniterealtime.org/t/openfire-4-6-4-is-released/90359), bringing some bug fixes.
+
+## Libraries
+
+[A Qt XMPP Bridge for Telegram based on QXmpp](https://invent.kde.org/lnj/qxbridge/)
+
+[XMPP Self Provisioning with Mastodon](https://github.com/mellium/fediverse-xmpp-onboarding)
```

Comment:
There is a [new ProtoXEP for for pre-auth key generation](https://xmpp.org/extensions/inbox/preauth-ibr.html), which would allow for fancy new uses, such as [...]

Reply:
Such information makes news way better! Thanks!
69e8a66